SETUP/smoketests/smoketests_configuration.sh

```diff
@@ -0,0 +1,129 @@
+# This configuration file is used for environment set up as part of
+# pageload smoketests run in Continuous Integration.
+
+_DB_SERVER=localhost
```

Comment:
I know we have a somewhat fake configuration for unit tests, does it make sense to align the configs and have 1 for all test scenarios?

Reply:
Yes, there is. And I'd like to try to merge them. See step 2 in my initial patchset comment. :)

Reply:
I've done the `source` trick for now and I'll see if I can just use the smoketest one outright.

Reply:
I ended up just replacing `SETUP/tests/ci_configuration.sh` with the changes I need, as it doesn't break any of the existing CI tests, and the smoketest version is closer to being correct.